src/content/docs/en/install/auto.mdx

```diff
@@ -13,7 +13,7 @@ Read our [step-by-step manual installation guide](/en/install/manual/) instead.
 :::
 #### Prerequisites
 
-- **Node.js** - `v18.14.1` or higher.
+- **Node.js** - `v18.17.1` or higher.
```

Comment:
Wonder if this should precise the full requirements, which is 18.17.1 or higher for 18, not 19, and 20.3.0 or higher for 20.

Reply:
Good point. If users are on Node.js `v19` now try to install Astro, they will get the engine warning

Reply:
I applied the change in [`3a32833` (#7838)](https://github.com/withastro/docs/pull/7838/commits/3a328331e8fc6791db495c94d9990496fc39de1a), I will let Sarah decide on the format of the sentence.